kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/Environment.java

```diff
@@ -32,7 +32,11 @@ private Environment() {
     /**
      * The kafka version default value
      */
-    public static final String KAFKA_VERSION_DEFAULT = "3.6.0";
+    public static final String KAFKA_VERSION_DEFAULT;
+
+    static {
+        KAFKA_VERSION_DEFAULT = determineKafkaVersion();
```

Comment:
It'd be slightly nicer to read the properties file once and use the result for both kafka and kroxylicious.   But as this is static code executed once at class load time, it doesn't matter too much.